.github/workflows/publishing.yml

```diff
@@ -158,6 +160,54 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.REPO_BOT_ACCESS_TOKEN }}
 
+      - name: Retrieve wheel assets
```

Comment:
Nit: For consistency and to make it a bit easier to reproduce any failures of this workflow, it might be nice to also retrieve the docker image assets here.

Reply:
Fixed as suggested: the new `Retrieve assets` step will also download docker image assets. Modified the `cudaq_hpc` job to load assets from artifacts rather than downloading them.